src/desktop/apps/auction/components/__tests__/DOM.jest.js

```diff
@@ -0,0 +1,125 @@
+import { test } from '../DOM'
+const { DOM } = test
+
+import AcceptConditionsOfSaleModal from 'desktop/apps/auction_support/client/accept_conditions_of_sale_modal.coffee'
+import mediator from 'desktop/lib/mediator.coffee'
+
+jest.mock('jquery', x => () => ({
+  foo: 'bar',
+}))
+jest.mock(
+  'desktop/apps/auction_support/client/accept_conditions_of_sale_modal.coffee'
+)
+jest.mock('desktop/components/credit_card/client/confirm_registration.coffee')
+jest.mock('desktop/lib/mediator.coffee', x => ({
+  trigger: jest.fn(),
+}))
+
+Object.defineProperty(window.history, 'replaceState', {
+  value: jest.fn(),
+})
+
+Object.defineProperty(window.location, 'assign', {
+  value: jest.fn(),
+})
+
```

Comment:
I would avoid `Object.defineProperty` operations unless its needed (typically for lower-level JS manipulation)

`global.history.replaceState = jest.fn()` or `window.history.location = jest.fn()` reads idiomatically.